patches/net/minecraft/server/MinecraftServer.java.patch

```diff
@@ -205,6 +217,18 @@
      }
  
      public SystemReport fillSystemReport(SystemReport p_177936_) {
+@@ -1135,6 +_,11 @@
+             "Enabled Feature Flags",
+             () -> FeatureFlags.REGISTRY.toNames(this.worldData.enabledFeatures()).stream().map(ResourceLocation::toString).collect(Collectors.joining(", "))
+         );
++        // Neo: Append modded feature flags to crash reports
++        p_177936_.setDetail(
++                "Enabled Feature Flags (Modded)",
```

Comment:
I'm not sure I like `(Modded)` here - could we potentially use `Enabled Mod Features` instead?